www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+---
+layout: page
+title: Introducing Karmashapes
+published: false
+date: 2023-06-19
+permalink: /blog/2023/06/19/introducting-karmashapes/
+category: blog
+excerpt: "Thanks to the Karmashapes initiative, Who’s On First now provides the best open data for towns and villages in India."
+authors: [Justin Meyers, stepps00, nvkelso]
+image: "images/coverage_overview.png)"
+tag: [whosonfirst,wof,data,import,india]
+---
+
+[![](images/coverage_overview.png)]()
```

Comment:
Update this image to a locally downloaded copy of the Cooper Hewitt image you caption and link to below?

Add link to the image in the final `()` of the markdown.